common.js

```diff
@@ -1728,6 +1728,521 @@ const allSoloCars = {
+      'all': ['bsp']
+    },
+    '280Z (incl. 2+2) - Alternate part: headlight covers': {
+      '1970': ['fp'],
+      '1971': ['fp'],
+      '1972': ['fp'],
+      '1973': ['fp'],
+      '1974': ['fp'],
+      '1975': ['fp'],
+      '1976': ['fp'],
+      '1977': ['fp'],
+      '1978': ['fp']
+    },
+    '280ZX': {
+      'all': ['bsp']
+    },
+    '280ZX (incl. 2+2) - Alternate part: headlight covers': {
+      '1979': ['fp'],
+      '1980': ['fp'],
+      '1981': ['fp'],
+      '1982': ['fp'],
+      '1983': ['fp']
+    },
+    '280ZX Turbo': {
+      'all': ['bsp']
+    },
+    '300ZX (non-turbo)': {
+      '1984': ['hs', 'stu', 'stx', 'esp'],
+      '1985': ['hs', 'stu', 'stx', 'esp'],
+      '1986': ['hs', 'stu', 'stx', 'esp'],
+      '1987': ['hs', 'stu', 'stx', 'esp'],
+      '1988': ['hs', 'stu', 'stx', 'esp'],
+      '1989': ['hs', 'stu', 'stx', 'esp'],
+      '1990': ['fs', 'esp'],
+      '1991': ['fs', 'esp'],
+      '1992': ['fs', 'esp'],
+      '1993': ['fs', 'esp'],
+      '1994': ['fs', 'esp'],
+      '1995': ['fs', 'esp'],
+      '1996': ['fs', 'esp']
+    },
+    '300ZX Turbo': {
+      '1984': ['fs', 'bsp'],
+      '1985': ['fs', 'bsp'],
+      '1986': ['fs', 'bsp'],
+      '1987': ['fs', 'bsp'],
+      '1988': ['fs', 'bsp'],
+      '1989': ['fs', 'bsp'],
+      '1990': ['cs', 'bsp'],
+      '1991': ['cs', 'bsp'],
+      '1992': ['cs', 'bsp'],
+      '1993': ['cs', 'bsp'],
+      '1994': ['cs', 'bsp'],
+      '1995': ['cs', 'bsp'],
+      '1996': ['cs', 'bsp']
+    },
+    '300ZX (Z31 chassis) - Alternate part: headlight covers': {
+      '1984': ['fp'],
+      '1985': ['fp'],
+      '1986': ['fp'],
+      '1987': ['fp'],
+      '1988': ['fp'],
+      '1989': ['fp']
+    },
+    '300ZX (Z32 chassis; non-turbo) - Alternate part: rear facing hood scoop (3.5” max height)': {
+      '1990': ['fp'],
+      '1991': ['fp'],
+      '1992': ['fp'],
+      '1993': ['fp'],
+      '1994': ['fp'],
+      '1995': ['fp'],
+      '1996': ['fp']
+    },
+    '310': {
+      'all': ['fsp']
+    },
+    '350Z NISMO': {
+      '2004': ['bs'],
+      '2005': ['bs'],
+      '2006': ['bs'],
+      '2007': ['bs'],
+      '2008': ['bs']
+    },
+    '350Z (non-NISMO)': {
+      '2002': ['bsp', 'stu'],
+      '2003': ['bsp', 'stu', 'fp', 'fs'],
+      '2004': ['bsp', 'stu', 'fp', 'fs'],
+      '2005': ['bsp', 'stu', 'fp', 'fs'],
+      '2006': ['bsp', 'stu', 'fp', 'fs'],
+      '2007': ['bsp', 'stu', 'fp', 'fs'],
+      '2008': ['bsp', 'stu', 'fp', 'fs'],
+      '2009': ['bsp', 'stu', 'fp', 'fs']
+    },
+    '370Z NISMO': {
+      '2009': ['bs', 'stu'],
+      '2010': ['bs', 'stu'],
+      '2011': ['bs', 'stu'],
+      '2012': ['bs', 'stu'],
+      '2013': ['bs', 'stu'],
+      '2014': ['bs', 'stu'],
+      '2015': ['bs', 'stu'],
+      '2016': ['bs', 'stu'],
+      '2017': ['bs', 'stu'],
+      '2018': ['bs', 'stu'],
+      '2019': ['bs', 'stu'],
+      '2020': ['bs', 'stu']
+    },
+    '370Z (non-NISMO)': {
+      '2003': ['stu', 'fp'],
```

Comment:
This model was only produced starting in 2009, please remove 2003 to 2008 from this model, thanks!